galleries/users_explain/figure/backends.rst

```diff
@@ -248,3 +249,71 @@ backend, use ``module://name.of.the.backend`` as the backend name, e.g.
 ``matplotlib.use('module://name.of.the.backend')``.
 
 Information for backend implementers is available at :ref:`writing_backend_interface`.
+
+.. _figures_not_showing:
+
+Debugging the figure windows not showing
+----------------------------------------
+
+Sometimes things do not work as expected, usually during an install.
+
+If you are using a Notebook or integrated development environment (see :ref:`notebooks-and-ides`),
+please consult their documentation for debugging figures not working in their
+environments.
+
+If you are using one of Matplotlib's graphics backends (see :ref:`standalone-scripts-and-interactive-use`), make sure you know which
+one is being used:
+
+.. code-block:: python3
+
+   import matplotlib
+
+   print(matplotlib.get_backend())
+
+Try a simple plot, as see if the GUI opens:
+
+.. code-block:: python3
+
+   import matplotlib
+   import matplotlib.pyplot as plt
+
+   print(matplotlib.get_backend())
+   plt.plot((1,4,6))
+   plt.show()
+
+If it does not, you perhaps have an installation problem.  A good step at this
+point is to ensure that your GUI toolkit is installed properly, taking
+Matplotlib our of the testing.  Almost all GUI toolkits have a small test
+program that can be run to test basic functionality.  If this test fails, try re-installing.
+
+PyQT
+^^^^
+
+.. code-block:: bash
+
+   python -c "from PyQt5.QtWidgets import *; app = QApplication([]); win = QMainWindow(); win.show(); app.exec()"
+
+If you have ``PySide`` or ``PyQT6`` installed instead, just change the import
+accordingly
+
+GTK
+^^^
+
+.. code-block:: bash
+
+   python3 -c 'from gi.repository import Gtk; win = Gtk.Window(); win.connect("destroy", Gtk.main_quit); win.show(); Gtk.main()'
```

Comment:
Should use double quotes outside and single quotes inside (rather than the other way round), because that'll also work on Windows.